pkg/config/config.go

```diff
@@ -35,14 +35,23 @@ var (
 	validRoles   = []string{"controlplane", "node"}
 )
 
+type OVNKubernetesConfig struct {
```

Comment:
Let's not make a separate tree of structs, and use the clusterconfig level as the ClusterCIDR and ServiceCIDR parameters; those must go at the same level